soc/renesas/smartbond/da1469x/power.c

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright (c) 2023 Renesas Electronics Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/init.h>
+#include <zephyr/pm/pm.h>
+#include <zephyr/logging/log.h>
+#include <zephyr/sys/barrier.h>
+#include <soc.h>
```

Comment:
soc.h is not needed here, please, include the necessary headers (maybe cmsis core for nvic?)